root/release/edit/tracklist.tt

```diff
@@ -324,6 +328,10 @@
     <!-- /ko -->
 
     <div data-bind="visible: loaded() && !collapsed()">
+      <p class="error field-error" style="padding: 1em" data-bind="showErrorWhenTabIsSwitched: needsFormat">
+        [%~ l('A format is required. Select the checkbox next to the input if you don’t know it. Make sure to check the top of the tracklist.') %]
```

Comment:
That seems to imply that the checkbox is at the top of the tracklist?

Reply:
Hm? It is at the top of the tracklist (and the previous wording implies the same thing).

Reply:
Oh, that may be confusing. What's actually meant to be said here is to check at the top of the tracklist for the *release format*. Maybe should make that more explicit?

Reply:
I agree, referring to labels (format/unknown) seems more user-friendly than just tag names (input/checkbox).

Reply:
I don't understand the distinction being made. The medium format ("input") is at the top of tracklist. The "I don't know" checkbox is next to the medium format, so it's also at the top of the tracklist.

I agree that referring to specific labels rather than inputs is better.